playbooks/common/openshift-cluster/upgrades/upgrade_control_plane.yml

```diff
@@ -25,6 +25,7 @@
       role: master
       local_facts:
         embedded_etcd: "{{ groups.oo_etcd_to_config | default([]) | length == 0 }}"
+        debug_level: "{{ openshift_master_debug_level | default(openshift.common.debug_level) }}"
```

Comment:
Updated but please double check my syntax, I didn't run a full HA upgrade after the change.